assets/web-ui/report.html

```diff
@@ -24,6 +24,15 @@ <h2>Initializing...</h2>
             </div>
             <div id="colorscheme-manager" class="widget" style="flex: 0 0 220px;"></div>
             <div id="category-legend" class="widget"></div>
+            <div id="search-container" class="widget" style="flex: 0 0 auto; padding-right: 5px;">
```

Comment:
This is a bit wide. I think we could, in order to save space:
- remove the search button
- make the X button as small as possible, it should ideally be placed in the text input